client/src/components/geneExpression/geneSet.js

```diff
@@ -84,14 +84,17 @@ class GeneSet extends React.Component {
       setGenes,
       setGenesWithDescriptions,
       isDiffExp,
```

Comment:
The existence of this flag breaks the model -- ie, "diff exp" results are not "just a gene set", but are now special in some way.  And overloading the semantics of the description field to be _both_ a description (user editable) and a flag will lead to fragility and likely future bugs (ie, user edits the field, and it no longer acts like a diffexp result).

I think the only solution is to pick one of:
* diff exp just generates a _generic_ gene set.  Get rid of the `isDiffExp` flag, including the presentation of p-val and lfc.
* Or, diffexp are "magically" special, and we need to add the flag and p-val/lfc to the gene set model, persisting them.

the latter seems untenable.   The former seems correct, and aligned with a related discussion @ambrosejcarr has been having around the (non)value of lfc/pval.

@signechambers1 - you also should comment.  The model as implemented is not consistent and will break if user edits the description.

Reply:
I support moving to `just a generic geneset` if @ambrosejcarr supports nonvalue of pval

Reply:
Talked with @seve and @MaximilianLombardo. To move forward, we are going to remove the pvalues and log fold change from the diff exp output to allow for diffexp to be a generic gene set in this PR. 

Ambrose and Max have both said these values may have limited value to the user so we may be able to remove these permanently. However, to guard against the situation where we remove these values and then our users are upset about this, @MaximilianLombardo is going to check in with a few people on active threads and get more community feedback before we decide what to do in the medium / longer term.